app/src/App.js

```diff
@@ -11,33 +12,20 @@ import Footer from './components/Footer';
 import ScrabbleTile from './components/ScrabbleTile';
 import ScrabbleHand from './components/ScrabbleHand';
 import ScrabbleBoard from './components/ScrabbleBoard';
+import { playersTiles } from './consts';
 
 function App() {
-  const [currentBoard, setCurrentBoard] = useState(Array(15).fill(Array(15).fill(null)));
 
-  useEffect(() => {
-    const fakeTile = { value: 'A', score: 1 };
-    setCurrentBoard(currentBoard => {
-      const newBoard = currentBoard.map((row, rowIndex) => {
-        return row.map((element, columnIndex) => {
-          if (rowIndex === 3 && columnIndex === 2) {
-            return fakeTile;
-          }
-          return element;
-        });
-      }
-      );
-      return newBoard;
-    });
-  }, []);
+  const { currentBoard, currentTile } = useContext(GameContext);
 
   return (
     <BrowserRouter>
       <div className="page-container">
         <div className="content-wrap">
           <Header />
           <ScrabbleTile value={'B'} score={2} />
-          <ScrabbleHand arrOfPlayersCurrentLetters={[{ value: 'C', score: 3 }, { value: 'D', score: 4 }, { value: 'C', score: 3 }, { value: 'D', score: 4 }, { value: 'C', score: 3 }, { value: 'D', score: 4 }, { value: 'C', score: 3 }, { value: 'D', score: 4 }]} />
+          {currentTile && `The current tile value is ${currentTile.value} and score is ${currentTile.score} id is ${currentTile.id}`}
+          <ScrabbleHand arrOfPlayersCurrentLetters={playersTiles} />
```

Comment:
Mostantól az elnevezésekbe is bele fogok kötni, bocsika.

Ez a prop `arrOfPlayersCurrentLetters` túlságosan bonyolult, hosszú nevű. Simán csak elnevezhetnéd `tiles`-nak is, hiszen mit csinál a komponens? Megjeleníti azokat a tile-okat amiket beadsz propnak